subfun/convertTsv2mat.m

```diff
@@ -8,7 +8,7 @@ function convertTsv2mat(tsvFile)
 % Read the tsv file
 t = spm_load(tsvFile);
 fprintf('reading the tsv file : %s \n', tsvFile)
-conds = t.trial_type;        % assign all the tsv information to a variable called conds.
+conds = t.condition;        % assign all the tsv information to a variable called conds.
```

Comment:
I adjusted according to the motion dataset, but this will change depending on how the tsv file is organized and what is the name of the columns/headers

Reply:
ok. Let me know.

Reply:
and I will adjust the tsv file creation accordingly.

Reply:
Yup the specs say to use `trial_type`: see the table at the top of this page.
https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html